Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN yum upgrade -y \
           pkgconfig \
           systemd-devel \
           zlib-devel \
-          nc
```

Comment:
Furthermore, do we really need to install `pkgconfig` and `-devel` packages? We don't build anything inside this container.

Reply:
So we started with this dockerfile: https://github.com/fluent/fluent-bit/blob/master/Dockerfile#L64

And then converted it to make it work on to amazon linux. I think there are some runtime dependencies needed. 

I am not an expert in this though and did it fairly hastily... making this image smaller has been on our backlog for a long time but we've never gotten to it. So thanks for helping.

I am fairly certain that `systemd-devel` is needed. I remember that there was an issue in the first version because I didn't include any runtime dependencies which broke the systemd/journald logs input plugin. Adding systemd to the image fixed it.

Reply:
I think `nc` can probably be removed. It was supposed to be part of a recommended healthcheck for FireLens but then we never published it. 

There's a very small chance that some agent functional tests were written assuming the command is there, and removing it might break those.

Reply:
If you think any of the other runtime dependencies can removed based on looking at the debian/distroless dockerfile let me know.